src/extension.ts

```diff
@@ -147,6 +148,10 @@ export async function activate(context: ExtensionContext) {
 		await sendCommandTrackingEvent(NewCamelSpringBootProjectCommand.ID_COMMAND_CAMEL_SPRINGBOOT_PROJECT);
 	}));
 
+	context.subscriptions.push(commands.registerCommand(NewCamelFileCommand.ID_COMMAND_CAMEL_NEW_FILE, async () => {
+		await new NewCamelFileCommand().create();
+	}));
```

Comment:
interested to have the commanTrackingEvent, it will allow to know if users are using the precise command directly or passing through the generic one more often

Reply:
in that case I will reenable command in palette by default because now it is hidden in command palette and available only under File> New File...